src/main/java/org/sunbird/portal/department/service/UserUtilityUtils.java

```diff
@@ -0,0 +1,73 @@
+package org.sunbird.portal.department.service;
+
+import java.io.IOException;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.UUID;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.sunbird.common.util.CbExtServerProperties;
+import org.sunbird.core.logger.CbExtLogger;
+import org.sunbird.core.producer.Producer;
+import org.sunbird.portal.department.model.Actor;
+import org.sunbird.portal.department.model.AuditEvents;
+import org.sunbird.portal.department.model.Context;
+import org.sunbird.portal.department.model.Edata;
+import org.sunbird.portal.department.model.LastLoginInfo;
+import org.sunbird.portal.department.model.ObjectData;
+import org.sunbird.portal.department.model.Pdata;
+import org.sunbird.portal.department.model.Rollup;
+import org.sunbird.workallocation.util.LastLoginConstants;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@Service
+public class UserUtilityUtils {
```

Comment:
Need to update the class name. Make sure that this is a utility class for posting telemetry events. Not only for user related actions. 
Need to move this to another new package which should be separate for telemetry related classes.